src/VM/Handler/Opcode8138Handler.cpp

```diff
@@ -21,12 +21,15 @@
 #include "../../VM/Handler/Opcode8138Handler.h"
 
 // C++ standard includes
+#include <vector>
 
 // Falltergeist includes
 #include "../../Logger.h"
 #include "../../VM/Script.h"
 #include "../../Game/ContainerItemObject.h"
 #include "../../Game/CritterObject.h"
+#include "../../Game/ItemObject.h"
+#include "../../Format/Enums.h"
```

Comment:
Not sure if it's appropriate to use File Format-related enums in our engine code. Basically everything under `Format` namespace should be used solely when loading files (via `ResourceManager`).